meta/dynamic-layers/msc-ldk-core/recipes-core/images/msc-image-qt5.bb

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2022 AVNET Embedded, MSC Technologies GmbH
```

Comment:
Why do you copy msc-image-qt5.bb from msc-ldk-core?

Reply:
I don't find msc-image-qt5.bb file in msc-ldk-core. so we have created one. I need to check in meta-msc-arm-extensions as not using it.

For now, we have used meta-arm latest directory which TI is using - https://github.com/anilpatelavnet/meta-arm